plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleStructureAssistant.java

```diff
@@ -121,6 +121,9 @@ public OracleStructureAssistant(OracleDataSource dataSource)
             if (ArrayUtils.contains(objectTypes, OracleObjectType.CONSTRAINT, OracleObjectType.FOREIGN_KEY) && objects.size() < maxResults) {
                 // Search constraints
                 findConstraintsByMask(session, schema, objectNameMask, objectTypes, maxResults, objects);
+                if(!containsOnlyConstraintOrFK(objectTypes)){

```

Comment:
I would suggest to add space before and after parenthesis
`if (!containsOnlyConstraintOrFK(objectTypes)) {`